Makefile

```diff
@@ -220,9 +221,12 @@ cni-plugins: get-cni-sources .out-stamp
 		"amazon/amazon-ecs-build-cniplugins:make"
 	@echo "Built amazon-ecs-cni-plugins successfully."
 
-run-integ-tests: test-registry gremlin container-health-check-image
+run-integ-tests: test-registry gremlin container-health-check-image run-sudo-integ-tests
 	. ./scripts/shared_env && go test -race -tags integration -timeout=7m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
 
+run-sudo-integ-tests:
+	. ./scripts/shared_env && sudo -E ${GO_EXECUTABLE} test -race -tags sudo -timeout=5m -v ./agent/engine/... ./agent/stats/... ./agent/app/...
```

Comment:
the only sudo test is in `./agent/engine`

You could get rid of the stats and app calls here, and then get rid of the !sudo tags in app and stats.

Reply:
done